source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -50,9 +51,16 @@ def get(self, request, instance, shape=None):
         if not self.external_api:
             raise TechnicalError('global autocomplete not configured')
 
-        url = '{endpoint}?q={q}&limit={count}'.format(endpoint=self.external_api,
-                                                      q=request['q'],
-                                                      count=request['count'])
+        params = {
```

Comment:
for me (it has not been done in your PR) we screwd on those parameters.

the `request` should only be the requested word to autocomplete (`request["q"]`)

And we should add another param with the `count` and the `from`

I think thus the `AbstractAutocomplete.get(..)` should be written with only the core parameters:
``` python
def get(self, q, instance, count, **kwargs):
```

and add the needed additional params in the specific class (for Autocomplete.Kraken the `search_type`, the `_current_datetime`, ...)

what do you think @kinnou02 @TeXitoi @pbougue ?